core-client/src/lib.rs

```diff
@@ -1799,12 +1814,22 @@ pub async fn execute_cmd(
                     )?;
                     core_endpoints_resp.insert(cur_core.party_id as u32, core_endpoint_resp);
 
+                    // NOTE: cur_role might not match the storage path, as a workaround,
+                    // if the kms config exists, then use the my_id field from there.
+                    let storage_role = match cur_core.config_path {
```

Comment:
Same comment as above

Reply:
answered in https://github.com/zama-ai/kms/pull/286#discussion_r2559925879